.gitignore

```diff
@@ -8,4 +8,5 @@ __pycache__/
 *~
 .tox
 .cache
-.eggs
\ No newline at end of file
+.eggs
+.vscode/settings.json

```

Comment:
We would like to refrain from adding editor specific ignore rules to the repository `.gitignore` file. Can you please add this to your local global `.gitignore` files instead?